README.md

```diff
@@ -50,6 +84,52 @@ sls deploy
 ```
 5. 👀 Head to the CloudWatch section of the AWS Console to check out your new dashboards 📊 and alarms ⏰ !
 
+
+## Getting Started with AWS SAM or CloudFormation
+
+This method uses the SLIC Watch _CloudFormation Macro_. It is very simple to add this macro as a transform to your SAM or CloudFormation template, but first you must ensure that the SLIC Watch Macro has been deployed to the same AWS account/region as your application.
+
+### Deploying the SLIC Watch Macro
+Deploy the SLIC Watch Macro in your account: 
+- _Method 1 using the Service Application Repository (SAR) console_: Go to [SLIC Watch in the Serverless Application Repository](https://serverlessrepo.aws.amazon.com/applications/eu-west-1/949339270388/slic-watch-app) and click the Deploy button.
+- _Method 2 (using SAR with CloudFormation)_: Add the SLIC Watch SAR application as a resource in a template:
```

Comment:
It is not very clear in the doc the **serverlessrepo-slic-watch-app** macro has to deployed before a SAM project.

Reply:
@direnakkoc Done - can you check the updates in the new commit and see if it is clear please?